packages/flutter/lib/src/widgets/app.dart

```diff
@@ -948,6 +951,25 @@ class WidgetsApp extends StatefulWidget {
   /// {@endtemplate}
   final Map<Type, Action<Intent>> actions;
 
+  /// {@template flutter.widgets.widgetsApp.restorationScopeId}
+  /// Providing a restoration ID enables state restoration for the app.
```

Comment:
there's a line above this, like, "The identifier to use when enabling state restoration for the navigator." or whatever, missing. (The first line of docs should be a noun phrase.)